chart/values.yaml

```diff
@@ -590,7 +590,31 @@ loki-stack:
 
 # Eventing component which enables or disables eventing-related components.
 eventing:
-  enabled: true
+  enabled: false
+  stats:
```

Comment:
sorry since we've discussed that stats are callhome specific now I think this should actually be inside callhome and not eventing.

Reply:
done. PTAL @tiagolobocastro